website/src/components/Homepage/Header/HomepageHeader.module.scss

```diff
@@ -1,67 +1,76 @@
+@import '@site/src/css/fonts';
+
 .heroBanner {
+  --ifm-heading-color: #fff;
+  --ifm-button-background-color: #fff;
+  --ifm-color-gray-900: #12161B;
+
   height: 725px;
   width: 100%;
-  padding: 4rem 0;
-  color: white;
+  padding: 54px 10px;
+  color: var(--ifm-heading-color);
   overflow: hidden;
-  background: rgb(21, 33, 107);
   background-image: url(@site/static/img/homepage/hero-image.jpg);
   background-position-x: center;
   background-position-y: top;
   background-repeat: no-repeat;
   background-size: cover;
-}
+  transition: height 0.3s ease;
+
+  h1 {
+    @include h2;
+    font-size: 120px;
+    line-height: normal;
+    letter-spacing: -6px;
+    margin-left: -10px;
+  }
 
-html[data-theme='dark'] {
-  --ifm-background-color: #202429;
-  --ifm-background-surface-color: #202429;
-  --ifm-footer-background-color: #202429;
-}
+  h2 {
+    @include t1;
+    max-width: 450px;
+    min-height: 60px;
+    margin-top: -30px;
+    margin-bottom: 32px;
+    letter-spacing: 0.1px;
+  }
+
+  .buttons {
+    height: 50px;
 
-.heroBanner h1 {
-  font-family: 'Nunito Sans', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 120px;
-  line-height: 150px;
-  margin-left: -10px;
-}
+    a {
```

Comment:
Isn't it possible to assign a dedicated CSS class for links (text) inside `.buttons`?

Then you wouldn't need CSS nesting.

Reply:
Fixed.